internal/processor/title.go

```diff
@@ -4,9 +4,11 @@ import (
 	"fmt"
 	"regexp"
 	"strings"
+
+	"github.com/autobrr/omegabrr/internal/domain"
 )
 
-func processTitle(title string) []string {
+func processTitle(title string, cfg *domain.ArrConfig) []string {
```

Comment:
With the change to titles.Add we can do the same to the parent function. Only value we are using is the boolean `MatchRelease` so this can be simplified to the following:

```golang
func processTitle(title string, matchRelease bool) []string {
```

Reply:
This one I think has to stay I think, otherwise they all error out since nothing can read from `github.com/autobrr/omegabrr/internal/domain`

Reply:
Hold that thought.

Reply:
Go should be able to remove or add imports as needed and if it's still needed somewhere, it's still needed :smile: